charts/values-overrides/internal/eks-dev-values.yaml

```diff
@@ -30,7 +30,7 @@ k8sDashboard:
 
 global:
   provisioner: ebs.csi.aws.com
-  tag: v2.40.27
+  tag: v2.42.4
```

Comment:
unfortunate these can't be a variable..

Reply:
You can in theory do it this way, you can use yq , maybe even envsub , I did it with yq on a gha one time